src/System.Azure.Experimental/System/Azure/Key.cs

```diff
@@ -13,14 +13,15 @@ namespace System.Azure.Authentication
     public static class Key {
         public static byte[] ComputeKeyBytes(string key)
         {
-            const int bufferLength = 128;
+            int size = key.Length * 2;
+            var buffer = size < 128 ? stackalloc byte[size] : (Span<byte>)new byte[size];
```

Comment:
nit: change var to Span<byte>. Removes the need for explicit cast.
```C#
Span<byte> buffer = size < 128 ? stackalloc byte[size] : new byte[size]; 
```

Reply:
Does not work today. @VSadov started a discussion  to enable it.

Reply:
Ah yes. I forgot:
https://github.com/dotnet/corefxlab/pull/1742/files#r137580133